x-pack/plugins/cases/public/components/user_actions/comment/comment.tsx

```diff
@@ -0,0 +1,371 @@
+/*
```

Comment:
This file is getting big. I can see some functions that can be extracted out to other file.

Reply:
It is new but all code got extracted from `helpers.tsx` and put it here.